pages/contact.html

```diff
@@ -23,12 +24,70 @@ <h2 class="underTitle">Big Dog Energy</h2>
             <a href="about_us.html">About Us</a>
             <a href="adopt_dog.html">Adopt a Dog</a>
             <a href="dogs_avail.html">Dogs Available</a>
-            <a class="active" href="success.html">Success Stories</a>
```

Comment:
you'll want to keep this in because it highlights the specific tab we're on from the navbar

Reply:
it's the wrong one :) the right on is Contact Us below this one